### PR TITLE
Add Consult Command

### PR DESCRIPTION
**Pull request by Charles1026**
consult feature combines appt and editing the upcoming appointment. creates a past appointment for today's date, and if the upcoming appointment is today, it consumes it.

### CODE REVIEW

**Comment by Charles1026** [comment]
This is same as the previous PR which I closed and reopened because Codecov wasn't working

File: src/main/java/seedu/address/logic/commands/AddCommand.java [hunk 1]
```diff
@@ -22,7 +22,7 @@ public class AddCommand extends Command {
 
     public static final String COMMAND_WORD = "add";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the address book.\n"
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a person to the app.\n"
```
Comment (kxrt) [inline src/main/java/seedu/address/logic/commands/AddCommand.java:25]: Good catch.

File: src/main/java/seedu/address/logic/parser/DeletePastAppointmentCommandParser.java [hunk 2]
```diff
@@ -12,7 +12,7 @@
 public class DeletePastAppointmentCommandParser implements Parser<DeletePastAppointmentCommand> {
     /**
      * Parses the given {@code String} of arguments in the context of the DeletePastAppointmentCommand
-     * and returns a DeleteCommand object for execution.
+     * and returns a DeletePastAppointmentCommand object for execution.
```
Comment (kxrt) [inline src/main/java/seedu/address/logic/parser/DeletePastAppointmentCommandParser.java:15]: Another good catch.

**Review by kxrt — APPROVED** [review]
Thanks for adding this stuff.

**Pull request merged**